muhasin/index.js

```diff
@@ -0,0 +1,42 @@
+/* for (let x = 0; x <= 0; x++) {
+    console.log(x)
+    console.log("x <=0 ==>", x <= 0); // true 
+    console.log("x <0 ==>", x < 0); // false
+} */
+
+// 1. 
+let determiner = 3;
```

Comment:
Here you can also another variable x and change the value of it depending on value of determiner